### PR TITLE
added deployment output 

### PR DESCRIPTION
**Pull request by vivishno**
-output of deployment set to action output.
-readme updated.

### CODE REVIEW

File: code/main.py [hunk 1]
```diff
@@ -90,7 +90,8 @@ def main():
         validate=client.deployments.validate(resource_group,"azure-sample",deployment_properties)
```
Comment (ashishonce) [inline code/main.py]: please change this deployment name to 
github repository name from which deployment is being made.

**Review by ashishonce — CHANGES_REQUESTED** [review]

File: code/main.py (hunk 1 above)
Comment (vivishno) [reply code/main.py]: updated

File: code/main.py [hunk 2]
```diff
@@ -16,6 +16,10 @@ def main():
     resource_group = os.environ.get("INPUT_RESOURCE_GROUP", default=None)
     mapped_params = os.environ.get("INPUT_MAPPED_PARAMS", default="{}")
     deployment_mode=os.environ.get("INPUT_DEPLOYMENT_MODE", default="Incremental")
+    repo_name=os.environ.get("GITHUB_REPOSITORY")
+    
+    repo_name=repo_name.split('/')
+    repo_name=repo_name[0]+'_'+repo_name[1]
```
Comment (ashishonce) [inline code/main.py]: use deployment_name

File: code/main.py [hunk 3]
```diff
@@ -87,20 +91,24 @@ def main():
...
         validate.wait()
     except Exception as ex:
-        raise ActionDeploymentError(ex)    
+        raise ActionDeploymentError(ex)
+    deployment_async_operation=None    
     try:
         deployment_async_operation = client.deployments.create_or_update(
                 resource_group,
-                'azure-sample',
+                repo_name,
```
Comment (ashishonce) [inline code/main.py]: deployment_name

**Review by ashishonce — CHANGES_REQUESTED** [review]

File: code/main.py (hunk 2 above)
Comment (vivishno) [reply code/main.py]: updated

**Review by ashishonce — APPROVED** [review]

**Pull request merged**